app/modules/conversations/conversation/conversation_service.py

```
@@ -520,7 +539,7 @@ async def _update_conversation_title(self, conversation_id: str, new_title: str)
         self.sql_db.commit()
 
     async def regenerate_last_message(
-        self, conversation_id: str, user_id: str, node_ids: List[NodeContext] = []
+        self, conversation_id: str, user_id: str, node_ids: List[NodeContext] = [], stream: bool = True
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)